.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  group: ci-${{ github.sha }}
+  cancel-in-progress: true
+
+permissions: {}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  lint:
```

Comment:
One final question (not sure if we answered it in the initial PR): Do we plan on splitting these up into individual files? I'd strongly vote for that

Reply:
I am not 100% sure either direction personally, as we roll out elsewhere. At this point for the dev-infra repo, I think we are okay to just have the single file for now.

I would like to defer to the decision to a little bit later.